### DISCUSSION TITLE
Hide application commands [feature request]

### DISCUSSION THREAD

**Discussion by AlexandrosGounis** [discordjs/discord.js#7603] [Q&A] [answered]
It is currently possible to `disable` application commands, given a role, or a user scope. 

However, this only dims down and freezes the options, but the users are still able to see the commands listed.

As admin-only commands are really useful, would it be possible to completely hide (remove from the slash list) some commands from the users and make them available to admins, (in the same Discord server)?

**Comment by Jiralite** [comment, marked as answer]
It is not possible. This is a feature request you need to make to Discord.

**Reply by AlexandrosGounis** [reply]
Isn't there a specific place where we do that? I suppose issues are not used for feature requests

**Reply by almostSouji** [reply]
Suggestions for the Discord Bot API go to <https://github.com/discord/discord-api-docs/discussions>
Though for this specific one: They're aware of it, will very likely be coming
